ReplicationAcceptance/ReplicationAcceptance/ReplicationAcceptance.m

```diff
@@ -1581,7 +1708,8 @@ -(void) testURLConnectionChangeTrackerWithRealRemoteUsingAuthorizer
     });
     
     while (!changeTrackerStopped) {
-        [NSThread sleepForTimeInterval:5.0f];
+        [[NSRunLoop currentRunLoop] runMode:NSDefaultRunLoopMode
+                                 beforeDate:[NSDate distantFuture]];
```

Comment:
Is there any reason why this doesn't sleep for 0.1s at a time like the other loops?